apps/pie-storybook/stories/introduction/component-versions.mdx

```diff
@@ -0,0 +1,46 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Introduction/Component versions" />
+
+# Component Versions
+
+An important feature of web components is that they must be registered in the DOM before they can be used. This tells the browser what component to use when it tries to parse a particular selector such as `<pie-button>`.
+
+## Table of Contents
+- [How we register components](#how-we-register-components)
+- [Version mismatch issues](#version-mismatch-issues)
+- [The `v` attribute](#the-v-attribute)
+
+## How we register components
+PIE web **components will register themselves automatically** when they are imported into the DOM. An illustration of what this looks like under the hood is like this:
+
+```js
+class MyComponent extends HTMLElement {
+    // Logic omitted for simplicity
+}
+
+// Register the component as a custom element. This tells the browser to use this class when it encounters the <my-component> tag in the DOM.
+customElements.define('my-component', MyComponent);
+```
+We have some safeguarding in place to ensure this registration only happens once. If you import the same component multiple times (which is easy to do in a complex application), you may see a warning in development mode stating that the component has already been registered.
+
+Consumers of PIE web components will never have to perform this step themselves, as it is handled automatically by the component library.
+
+## Version mismatch issues
+
+One of the most common issues when using PIE web components is when multiple versions of the same component are registered in the DOM. This can happen if you have multiple versions of the component library installed, or if you import the same component multiple times in your application.
```

Comment:
I'd maybe say something like "one possible issue when using PIE web components", otherwise it makes it sound like loads of people who are using the components are regularly encountering problems. What do you think?